test/test_other.py

```diff
@@ -92,6 +92,20 @@ def metafunc(self, wasmfs):
   return metafunc
 
 
+def also_with_wasm_longjmp(f):
+  def metafunc(self, wasmfs):
```

Comment:
Why `wasmfs`? Is this related to WasmFS?

Reply:
copy pasta. fixed.